Makefile

```diff
@@ -85,18 +86,32 @@ $(LUCENE_ANALYZER_JAR):
 
 jars: $(INDEXER_JAR) $(TAGSOUP_JAR) $(LUCENE_ANALYZER_JAR) $(LUCENE_CORE_JAR)
 
-docs/index.html: $(SOURCES)
-	mkdir -p docs
+webhelp/index.html: $(SOURCES)
```

Comment:
The reason it was `docs` is that I could not find the way to change it in first place, but I would have preferred a meaningful name.
I can put localsearch in `docs-localsearch` and keep the `webhelp` and `webhelp-localsearch` targets, what do you think is the best option?